Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM cirrusci/flutter:v1.12.13-hotfix.8-web
+
+WORKDIR /app
+
+COPY ./app_feup/ .
+
+RUN sudo chown -R cirrus:cirrus /app
+
+RUN flutter pub get
+
+CMD [ "sh", "-c" , "flutter analyze --no-pub --preamble . && flutter test"]
```

Comment:
Do we really want this `CMD` to run everytime we build the docker image? Also, if we are running this on the Dockerfile, isn't `travis.yml` really just running this twice? Once in the `docker build` command and another in the `docker run`?

Reply:
The CMD does not run with `docker build`. It is used as the default command for when it is not changed with `docker run <command>`.